### DISCUSSION TITLE
Spanish Translation

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#320] [Team Posts]
Dear all,

Luca Riva has completed the translation to Spanish in record time. Spanish will be a fully supported language from the next release on.

**Thank you, Luca!**

Best wishes from the TGV to London,

Stefan.

**Comment by TTC70** [comment]
...the team is getting bigger and better.
Thanks/congrats to Luca (who increase the percentage of Italians in the
project 😎😎).
Have a safe trip to London.
Antonio

Il Gio 16 Feb 2023, 08:43 Stefan Kebekus ***@***.***> ha
scritto:

**Comment by Lukesw74** [comment]
Thank you Stefan, Antonio ..
Ciao
Luca
